File: API/ICCMS-API/Models/QuotationItem.cs
using Google.Cloud.Firestore;

namespace ICCMS_API.Models
{
    [FirestoreData]
    public class QuotationItem
    {
        [FirestoreProperty("name")]
        public string Name { get; set; } = string.Empty;

        [FirestoreProperty("quantity")]
        public double Quantity { get; set; }

        [FirestoreProperty("unitPrice")]
        public double UnitPrice { get; set; }

        [FirestoreProperty("taxRate")]
        public double TaxRate { get; set; }

        [FirestoreProperty("lineTotal")]
        public double LineTotal { get; set; }
    }
}

File: Mobile/app/src/main/java/com/example/iccms_mobile/data/models/QuotationItem.kt
package com.example.iccms_mobile.data.models

import com.google.gson.annotations.SerializedName

data class QuotationItem(
    @SerializedName("itemId")
    val itemId: String = "",

    @SerializedName("name")
    val name: String = "",

    @SerializedName("description")
    val description: String = "",

    @SerializedName("quantity")
    val quantity: Int = 0,

    @SerializedName("unitPrice")
    val unitPrice: Double = 0.0,

    @SerializedName("totalPrice")
    val totalPrice: Double = 0.0
)
